pkgs/by-name/it/itstool/package.nix

```diff
@@ -16,16 +17,17 @@ stdenv.mkDerivation rec {
 
   strictDeps = true;
 
+  postPatch = ''
+    # Do not let autoconf find Python, but set it directly. This fixes cross-compilation.
+    sed -i '13d;20d' configure.ac
```

Comment:
I'm not comfortable with deleting two lines like that. I'd rather se a patch or `substituteInPlace --replace-fail` -- something with enough context so it cannot trivially continue when it should have failed.

Reply:
That's fail, I'll update the PR after I've tested the change locally.